.storybook/preview-head.html

```diff
@@ -0,0 +1 @@
+<script src="http://localhost:8098"></script>
```

Comment:
This is for the Vue dev tools?

Reply:
yeah, unrelated to this work, sorry

Reply:
No worries. It'd be helpful to reference it and link to the instructional post in the README whenever you do add it.